### PR TITLE
feature: add option to disable nginx beautifier

### LINKED ISSUE #936: add option to disable nginx beautifier
e.g. `DISABLE_NGINX_BEAUTIFIER`;

blocks like this:
```
        location / {
            access_by_lua_block {
                local http = require("resty.http")

                local target_ip = "192.168.0.1"
                local target_port = 8080
                local mac_address = "aa:bb:cc:dd:ee:ff"

                local httpc = http.new()
                httpc:set_timeout(1000)  -- 1 second timeout

                local res, err = httpc:request_uri("http://" .. target_ip .. ":" .. target_port, {method = "HEAD"})

                if not res then
                    ngx.log(ngx.NOTICE, "Target is not responding, attempting to send WoL packet")
                    local command = string.format("wakeonlan %s", mac_address)
                    local success = os.execute(command)
                    if success then
                        ngx.log(ngx.NOTICE, "Sent WoL packet to " .. mac_address)
                        ngx.sleep(30)
                    else
                        ngx.log(ngx.ERR, "Failed to send WoL packet")
                    end
                else
                    ngx.log(ngx.NOTICE, "Target is already responsive")
                end
            }

            proxy_pass http://192.168.0.1:8080;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection $connection_upgrade;
        }

```

cause:
```
npmplus  | /usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/nginxbeautifier.js:142
npmplus  |         return input.match(catchRegex)[1];
npmplus  |                                       ^
npmplus  | 
npmplus  | TypeError: Cannot read properties of null (reading '1')
npmplus  |     at extractTextBySeperator (/usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/nginxbeautifier.js:142:39)
npmplus  |     at extractAllPossibleText (/usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/nginxbeautifier.js:164:29)
npmplus  |     at strip_line (/usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/nginxbeautifier.js:193:35)
npmplus  |     at Object.clean_lines (/usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/nginxbeautifier.js:225:50)
npmplus  |     at Object.<anonymous> (/usr/local/share/.config/yarn/global/node_modules/nginxbeautifier/index.js:235:38)
npmplus  |     at Module._compile (node:internal/modules/cjs/loader:1358:14)
npmplus  |     at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
npmplus  |     at Module.load (node:internal/modules/cjs/loader:1208:32)
npmplus  |     at Module._load (node:internal/modules/cjs/loader:1024:12)
npmplus  |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
npmplus  | 
npmplus  | Node.js v20.13.1
```

### PR DESCRIPTION
**Pull request by realies**
fixes #936

### CODE REVIEW

File: compose.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ services:
 #      - "PHP83=true" # Activate PHP83, default false
 #      - "PHP83_APKS=php83-curl php83-openssl" # Add php extensions, see available packages here: https://pkgs.alpinelinux.org/packages?branch=v3.20&repo=community&arch=x86_64&name=php83-*, default none, requires PHP83
 #      - "PHP_APKS=php-pecl-apcu php-pecl-redis" # Add php extensions, see available packages here: https://pkgs.alpinelinux.org/packages?branch=v3.20&repo=community&arch=x86_64&name=php-*, default none, requires PHP82 and/or PHP83, not recommended, please use PHP82_APKS or PHP83_APKS
+#      - "DISABLE_NGINX_BEAUTIFIER=true" # disables nginxbeautifier, useful when it fails parsing non-standard configs
```
Comment (Zoey2936) [inline compose.yaml]: please move it up to the other "DISABLE_" options and also mention the default option. And please also set the default in the Dockerfile

File: rootfs/usr/local/bin/start.sh [hunk 2]
```diff
@@ -883,7 +883,7 @@ elif [ "$FULLCLEAN" = "true" ]; then
     rm -vrf /data/etc/goaccess
 fi
 
-nginxbeautifier -s 4 -r /data/nginx
+if [ -n "$DISABLE_NGINX_BEAUTIFIER" ]; then nginxbeautifier -s 4 -r /data/nginx; fi
```
Comment (Zoey2936) [inline rootfs/usr/local/bin/start.sh]: Can you please make the if check multi-line and the condition like all other true/false conditions and a syntax check fir this env

**Review by Zoey2936 — CHANGES_REQUESTED** [review]
Please see my comments, thanks

File: compose.yaml (hunk 1 above)
Comment (realies) [reply compose.yaml]: these are not really clustered together, but i moved it up

File: rootfs/usr/local/bin/start.sh (hunk 2 above)
Comment (realies) [reply rootfs/usr/local/bin/start.sh]: sure

**Review by Zoey2936 — APPROVED** [review]

**Pull request merged**